src/mumble/AudioConfigDialog.cpp

```diff
@@ -652,6 +623,39 @@ AudioOutputDialog::AudioOutputDialog(Settings &st) : ConfigWidget(st) {
 	qcbLoopback->addItem(tr("Server"), Settings::Server);
 
 	qcbDevice->view()->setTextElideMode(Qt::ElideRight);
+
+	// Distance in cm
+	qsMinDistance->setRange(0, 200);
+	// Distance in m
+	qsbMinimumDistance->setRange(0.0, 50.0);
+	// Distance in cm
+	qsMaxDistance->setRange(10, 2000);
+	// Distance in m
+	qsbMaximumDistance->setRange(1.0, 1000.0);
+	qsMinimumVolume->setRange(0, 100);
+	qsbMinimumVolume->setRange(qsMinimumVolume->minimum(), qsMinimumVolume->maximum());
+	qsBloom->setRange(0, 75);
+	qsbBloom->setRange(qsBloom->minimum(), qsBloom->maximum());
+
+	QString minDistanceTooltip = tr("Distance at which audio volume from another player starts decreasing");
+	QString maxDistanceTooltip = tr("Distance at which a player's audio volume has reached its minimum value");
+	QString minVolumeTooltip =
+		tr("The minimum volume a player's audio will fade out to with increasing distance. Set to 0% for it to fade "
+		   "into complete silence for a realistic maximum hearing distance.");
+	QString bloomTooltip = tr("Factor for sound volume increase");
```

Comment:
We need a better tooltip for the Bloom setting. This one was overtaken from what it currently is but I think it is not helpful at all.

@davidebeatrici @Kissaki can you think of a somewhat short-ish and concise (understandable) explanation of what Bloom is about?

Reply:
What does it actually do? How does the factor influence sound volume?

Reply:
Davide try to explain it to me on IRC not too long ago, but I think I only understood it partly. In some sense blooming appears to be that even if the audio is coming entirely e.g. from the right, then if it is loud enough you will also hear it through your left speaker.
So I guess the factor here is related to that loudness threshold somehow...

Reply:
here's what the comment in the code says:
https://github.com/mumble-voip/mumble/blob/260215ade90adb13bd2f83eba477ae85526db9ca/src/mumble/AudioOutput.cpp#L93-L95

Reply:
Is the closeness a change trigger, or is there a period where it goes towards max bloom the closer you get? On what closeness does the bloom become active?

Reply:
I would be fine improving the bloom description in a followup PR.

Reply:
> Is the closeness a change trigger, or is there a period where it goes towards max bloom the closer you get? On what closeness does the bloom become active?

Bloom starts fading in once the audio source enters the minimum distance at which audio volume starts decaying (the min. dist. setting) and then ramps up linearly until the audio source sits on top of the player